templates/admin-section.php

```diff
@@ -0,0 +1,84 @@
+<?php
```

Comment:
I think we should renamed this file to admin-settings.php, since section, in Nextcloud terms wouldn't correctly describe it. This is a template for displaying some settings inside a section, not the section itself.

Reply:
Renamed.